### ISSUE TITLE
Normalization of gluc and cholesterol.

### ISSUE THREAD

**Issue by carbasemin** [Jppbrbs/fcc-medical-data-visualizer#1]
In line34-37, normalization:
```
df['cholesterol'] = (df['cholesterol'] > 1).astype(int)
# print(df['cholesterol'])
df['gluc'] = (df['gluc'] > 1).astype(int)
```
This doesn't make any sense to me. The goal is to set the values that are equal to 1 to 0 and the rest to 1. How is that accomplished here? 

This is the way I did it:
```
df.loc[df["cholesterol"] == 1, "cholesterol"] = 0
df.loc[df["cholesterol"] > 1, "cholesterol"] = 1
df.loc[df["gluc"] == 1, "gluc"] = 0
df.loc[df["gluc"] > 1, "gluc"] = 1
```

[Full project](https://github.com/carbasemin/freeCodeCamp/blob/master/data_analysis_with_python/projects/medical_data_visualizer.py) if you like.

**Comment by Jppbrbs** [comment]
Dear Emin,

The 'astype(int)' command works complementing the restriction written just before. 

When I wrote  "(df['cholesterol']>1).astype(int)"  the first part of this commands evaluates the values from the Cholesterol column and returns them as booleans (True or False), depending if the value respects the restriction or not.

After that, the astype(int) command converts it to integers values, by making True equal to 1 and False equal to 0.

It was just way I thought of doing it in less lines of code.

Below there are some pictures that simplifies what I explained. The first one, relates to the code   "(df['cholesterol']>1)"  without  ".astype(int)". The other one is the result of both commands complementing each other.


<img width="374" alt="df 'cholesterol'" src="https://user-images.githubusercontent.com/46424913/89919980-48402580-dbd2-11ea-9029-46f03ee0cf9b.png">

<img width="374" alt="df 'cholesterol' astype(int)" src="https://user-images.githubusercontent.com/46424913/89920034-59893200-dbd2-11ea-8020-647be861eb75.png">
